Exercise_C.js

```diff
@@ -13,6 +13,29 @@ If the number is divisible by 3 and 5 it should log Hong Kong.
 See README.md if you need help running your code.
 */
 
-for (let i = 1; i <= 30; i++) {
-  // CODE HERE
+let x = [
+  1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22,
+  23, 24, 25, 26, 27, 28, 29, 30,
+];
+
+function findhk() {
+
+  y = []
+  for (let i = 1; i <= 30; i++) {
+    if (x[i] % 3 == 0 && x[i] % 5 == 0) {
+     y.push("Hong Kong")
+    }
+    else if (x[i]%5 == 0) {
+     y.push("Kong")
+    }
+    else if (x[i]&3 == 0) {
+     y.push("Hong")
+    }
+    else {
+     y.push(i)
+    }
+  }
+  return y
 }
+
+console.log(findhk(x))

```

Comment:
Hello Alex 
This is an interesting way of solving this exercise which does work but an easier way would be this 
`for (let i = 1; i <= 30; i++) {
  if (i % 15 == 0) {
    console.log("Hong Kong");
  } else if (i % 3 == 0) {
    console.log("Hong");
  } else if (i % 5 == 0) {
    console.log("Kong");
  } else {
    console.log(i);
  }
}`